site/accounts/staywelcome.php

```diff
@@ -0,0 +1,11 @@
+<?php if(!defined('KIRBY')) exit ?>
+
+username: staywelcome
+password: >
+  $2a$10$LhoBWrjK9kPdmbD7/CsZVOopiTTpDRAiLCMrtucCtL5.P0O.OyIYm
```

Comment:
looks like it is salted https://getkirby.com/docs/toolkit/api/password/hash and length/entropy looks also quite good. I think this shouldn't be an issue...